### PR TITLE
ENHANCEMENT Add fluent icon to translatable fields

### PR DESCRIPTION
**Pull request by raissanorth**
Addresses one of the issues raised in #311 by adding a Fluent icon to translatable fields.

Design mockup this PR is based on: https://projects.invisionapp.com/share/ZBFVZJCKU38#/screens/279729292_X_Fluent

The translatable font icon that is used was pushed to the admin repo in the https://github.com/silverstripe/silverstripe-admin/commit/4bee4bbf8e731ebc7b6aeb79e1ba8451de87eec8 commit.

### CODE REVIEW

File: client/src/styles/fluent.scss [hunk 1]
```diff
@@ -88,3 +88,12 @@ span.badge.status-fluenthidden {
   text-shadow: none;
 }
 
+.fluent__localised-field .font-icon-translatable {
+  color: #6610F2;
```
Comment (robbieaverill) [inline client/src/styles/fluent.scss]: Could we use a variable here? If you wait until #362 is merge then you can use `$fluent-color-localised`

Comment (raissanorth) [reply client/src/styles/fluent.scss]: Sure, I am waiting patiently. ;)

**Comment by NightJar** [comment]
#363 is merged now :)

File: src/Extension/FluentExtension.php [hunk 2]
```diff
@@ -140,6 +143,14 @@ class FluentExtension extends DataExtension
      */
     private static $frontend_publish_required = true;
 
+    /**
+     * Controls whether the message indicating whether you are editing in the default locale is disabled
+     *
+     * @config
+     * @var bool
+     */
+    private static $disable_current_locale_message = false;
```
Comment (robbieaverill) [inline src/Extension/FluentExtension.php]: We aren't actually using this any more - can we remove it?

Comment (raissanorth) [reply src/Extension/FluentExtension.php]: Done.

**Pull request merged**